lib/client/signalflow/request_manager.js

```diff
@@ -390,11 +390,34 @@ function RequestManager(options) {
     //destroy all open SSE connections
   }
 
+  function registerLiveTail(callbacks, overrideRequestId ){
+    var requestId = overrideRequestId || getRequestIdAndIncrement();
+    knownComputations[requestId] = {
+      callback: callbacks,
+      onMessage: router.onMessage,
+      transport: transports.WEBSOCKET,
+      requestId: requestId,
+      lastDataBatchTime: router.getLatestBatchTimeStamp,
+    };
+    var jobObject = {};
+    jobObject.channel = requestId;
+    activeSocket.send(JSON.stringify(jobObject));
+  }
+
+  function unRegisterLiveTail(requestId){
+    activeSocket.close();
```

Comment:
Don't close the socket! There may be other live tails or signaflow jobs using it. The RequestManager does not own the lifecycle of the WebSocket (the `SignalflowClient` does).

What you want to do instead is sent a `stoplivetail` request through the WebSocket.